.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+    - id: check-added-large-files
+    - id: mixed-line-ending
+      args: ['--fix', 'no']
+      exclude: |
+         (?x)(
+            .*\.cmd
```

Comment:
>.*\.cmd [](http://example.com/codeflow?start=12&length=7)

We also have .md, .jpg, .png,, etc. that need to be excluded

Reply:
Should be resolved by new commit. If not then reopen.